plugins/tiddlywiki/codemirror/engine.js

```diff
@@ -69,6 +69,10 @@ function getCmConfig() {
 						value = tiddler.fields.text.trim();
 						config[element] = value;
 					break;
+					case "integer":
+					  value = parseInt(tiddler.fields.text.trim());
```

Comment:
Important: There needs to be the base parameter `parseInt(var, 10)` otherwise we will have problems!